numpy/_core/src/umath/loops_exponent_log.dispatch.c.src

```diff
@@ -1370,7 +1375,11 @@ NPY_NO_EXPORT void NPY_CPU_DISPATCH_CURFX(@TYPE@_ldexp)
 (char **args, npy_intp const *dimensions, npy_intp const *steps, void *NPY_UNUSED(func))
 {
 #ifdef SIMD_AVX512_SKX
-    if (IS_BINARY_SMALL_STEPS_AND_NOMEMOVERLAP) {
+    if ((npyv_loadable_stride_@suffix@(steps[0])) &&
+        (npyv_storable_stride_@suffix@(steps[1])) &&
+        (npyv_storable_stride_@suffix@(steps[2])) &&
```

Comment:
Thanks, now I guess this is annoying to test, but I think the AVX512 code needs to deine not just the current:
```
#define NPY_SIMD_MAXLOAD_STRIDE32  (0x7fffffff / 16)
```
but also the `64` suffixed version for this to be right for extremly large strides (i.e. reject them).  TBH, that seems fine to me though (would allow to use the i32 steps always, but that doesn't matter here).

Reply:
I added the NPY_SIMD_MAX*_STRIDE64 macro's. Its set to go now. I am not too worried about what exact values we use. They are fairly large anyways. It is annoying to test, apart from adding another dedicated CI  to run in an SDE, it will get tested in a few patches. Failures should show up.